PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Thanks for helping us make a better Snapd!
```

Comment:
Agree with @pedronis. Should also s/Snapd/snapd/.

LGTM with the version or something similar to it:

---
Thanks for helping us make a better snapd!

Have you signed the [license agreement](https://www.ubuntu.com/legal/contributors) and read the [contribution guide](CONTRIBUTING.md)?